x/ibc/23-commitment/types/merkle.go

```diff
@@ -156,26 +158,31 @@ func (MerkleProof) GetCommitmentType() exported.Type {
 // VerifyMembership verifies the membership pf a merkle proof against the given root, path, and value.
 func (proof MerkleProof) VerifyMembership(root exported.Root, path exported.Path, value []byte) error {
 	if proof.IsEmpty() || root == nil || root.IsEmpty() || path == nil || path.IsEmpty() || len(value) == 0 {
-		return errors.New("empty params or proof")
+		return sdkerrors.Wrap(ErrInvalidProof, "empty params or proof")
 	}
 
-	runtime := rootmulti.DefaultProofRuntime()
-	return runtime.VerifyValue(proof.Proof, root.GetHash(), path.String(), value)
+	if !ics23.VerifyMembership(proof.Spec, root.GetHash(), proof.Proof, []byte(path.String()), value) {
```

Comment:
Is the `key` in the confio `VerifyMembership` function the path? Or is it the last element of the path?

Reply:
I assume it's the full path